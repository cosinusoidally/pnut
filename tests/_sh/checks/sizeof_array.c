// expect_comp_failure
typedef int arr[1000000];

void main() {
  int a = sizeof(arr);
}
