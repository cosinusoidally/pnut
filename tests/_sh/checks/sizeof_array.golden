tests/_sh/checks/sizeof_array.c:7:0  comp_rvalue_go: sizeof is not supported for this type or expression
